examples/SigningWithAHDWalletPrivateNetworkCompatible.ts

```diff
@@ -1,5 +1,5 @@
-import {ExtendedKey, Network, Wallet} from 'nem2-hd-wallets';
-import {Account, Deadline, EmptyMessage, NetworkType, TransferTransaction} from "nem2-sdk";
+import {ExtendedKey, Network, Wallet} from 'symbol-hd-wallets';
+import {Account, Deadline, EmptyMessage, NetworkType, TransferTransaction} from "symbol-sdk";
```

Comment:
It's not related with rebranding, but note that the project uses in some parts import with simple quotes (e.g. from 'symbol-hd-wallets') whereas in others with double quotes (e.g. from "symbol-sdk").

Reply:
good catch, seems to be inside examples/ :+1: will fix.